demos/src/demo.scss

```diff
@@ -1,18 +1,13 @@
-$o-labels-is-silent: false;
 
-@import "../../main";
+$o-label-is-silent: false;
+$o-typography-is-silent: false;
+
 @import "o-typography/main";
 
+@import "../../main";
+
 @include oFontsIncludeAll();
 
 body {
 	margin: 1em;
```

Comment:
Probably better to remove `oFontsIncludeAll();` and include via demo dependancies along with o-normalise? i.e. https://github.com/Financial-Times/o-table/blob/master/origami.json#L28
I don't think including `o-typography` as a demo dependancy is needed though?